edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -0,0 +1,74 @@
+Feature: patron tests
+
+  Background:
+    * url baseUrl
+    * callonce login testAdmin
+    * configure headers = {  'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+    * def uuid = call random_uuid
+    * def firstName = call random_string
+    * def lastName = call random_string
+    * def owner = random_string
+    * def amount = call random_numbers
+    * def status = 'Available'
+    * def requestId = call random_uuid
+    * def servicePointId = call random_uuid
+
+  Scenario: Return total fees/fines regardless of fee/fine being attached to an item for a patron.
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem') { itemId: 388830d5-95db-4528-95b6-4ec9d37d4086, materialTypeId: 388830d5-95db-4528-95b6-4ec9d37d4087,materialTypeName: TestMaterial, itemBarcode: 11111}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4056, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4086,materialTypeName: TestMaterial1, itemBarcode: 2222}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4089, itemId: 388830d5-95db-4528-95b6-4ec9d37d4056 }
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 188830d5-95db-4528-95b6-4ec9d34d4086, itemId: 388830d5-95db-4528-95b6-4ec9d37d4086 }
```

Comment:
The description mentions `regardless of fee/fine being attached to an item`. That suggests there are multiple scenarios for the fees / fines to consider in this. Is that the case?

The two examples in this test seem to both be item related fee / fines.

Reply:
I guess the test says , return total amount of fine . PFB api response


  "charges": [
  ],
  "loans": [
  ],
  "totalCharges": {
    "amount": 1796518.0,
    "isoCurrencyCode": "USD"
  },
  "totalLoans": 0,
  "holds": [
  ],
  "totalChargesCount": 2,
  "totalHolds": 0
}

Reply:
can user be charged for other actions also apart from items related actions?

Reply:
> can user be charged for other actions also apart from items related actions?

I don't know. The name suggests so. I suggest asking the PO for more details about this test case.

Reply:
Thanks for feedback, added 1 more scenario for "printing cards"

Reply:
How does that help with the question about this scenario?

Reply:
previously both fine scenario were related to the item only, updated one has one more scenario which is **charges for printing cards**.

The test is expecting all the charges in response included with a patron .

Reply:
Are you referring to this line:
```
    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndCharges')
```

Reply:
yes

Reply:
Ok, when you said scenario above, I thought you meant a separate scenario rather than a step in this scenario. Apologies for misunderstanding.